docker/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:18.04
```

Comment:
Would it be possible to use a lightweight linux image such as alpine, instead of ubuntu?